index.js

```diff
@@ -172,20 +254,26 @@ const handleRequest = async event => {
         }
     }
 
+    // Check the cache service for data first - If cached data exists, return it
+    const cachedResponse = await checkCache(json);
```

Comment:
Does it matter that we're getting the cached response here even though the following logic depends on url.pathname for response logic?